Modules/Core/Transform/include/itkResampleInPlaceImageFilter.h

```diff
@@ -52,17 +49,17 @@ namespace itk
  * The purpose of this code is to generate the new origin and direction
  * that will remove the need for using the transform.
  *
- * Given a set of PhysFixedImagePoints (i.e. from the fixedImage space)
- * those points are converted to PhysMovingImagePoints = TfmF2M( PhysFixedImagePoints )
- * and then MovingContinuousIndexPoints = movingImage->TransformPhysicalPointToContinuousIndex( PhysMovingImagePoints )
- * to get image values.
+ * Given a set of PhysicalFixedImagePoints (i.e. from the fixedImage space)
+ * those points are converted to PhysicalMovingImagePoints = TfmF2M( PhysicalFixedImagePoints )
```

Comment:
IMO using capital letters for a variable name is misleading.

I think the term `TfmF2M` has not been introduced previously/it would help to explain what it means.